src/pages/Autolending/AutolendingWho.vue

```diff
@@ -17,30 +17,22 @@
 
 				<kv-lightbox
 					class="autolending-who-lightbox"
-					:no-padding-bottom="true"
-					:no-padding-top="true"
 					:visible="showLightbox"
+					title="Choose who you’ll support"
 					@lightbox-closed="showLightbox = false; showSelectedFilterOptions = false;"
 				>
-					<template slot="title">
-						<div class="who-titles-wrapper">
-							<transition :name="slideTransition" mode="out-in">
-								<h3 v-if="!showSelectedFilterOptions">
-									Choose who you’ll support
-								</h3>
-								<a
-									v-if="showSelectedFilterOptions"
-									role="button"
-									class="back-to-options"
-									@click.prevent="backToAllOptions"
-								>
-									<kv-icon class="arrow back-arrow" name="small-chevron" :from-sprite="true" />
-									Back to all options</a>
-							</transition>
-							<hr>
-						</div>
-					</template>
 					<div class="who-inputs-wrapper">
+						<transition :name="slideTransition" mode="out-in">
```

Comment:
This is not the same as the lines removed in the slot. When the transition happens, the title should be replaced with the "back to all options" with a slide transition